internal/gps/result.go

```diff
@@ -14,6 +14,14 @@ import (
 // additional methods that report information about the solve run.
 type Solution interface {
 	Lock
+	// The name of the ProjectAnalyzer used in generating this solution.
+	AnalyzerName() string
+	// The version of the ProjectAnalyzer used in generating this solution.
+	AnalyzerVersion() int
+	// The name of the Solver used in generating this solution.
+	SolverName() string
```

Comment:
I have no actual plan to do so, but it's possible that circumstances could demand it. We definitely needed the solver version field; as long as I was adding one, it seemed only to make sense to add the other.

Also...I wanted to make the `cdcl` thing explicit, because people would ask questions. Like you did. And then maybe get interested in working on the solver 😄